### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue #118 
Added missing social profiles

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
Please merge thanks!

**Comment by arjunpandey2003** [comment]
@mariobehling can you please merge this request also when i got the code a huge part was commented so should i remove the comments. Thanks

File: index.html [hunk 1]
```diff
@@ -577,7 +578,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://togaware.com/about/graham-williams/"><i class="icon fa fa-external-link"></i></a>
-
+											<a target="default" href="https://conferences.oreilly.com/strata/strata-sg-2016/public/schedule/.../253496"><i class="icon fa fa-external-link"></i></a>
```
Comment (vaibhavsingh97) [inline index.html]: This link welcomes me with 404 Error

**Comment by arjunpandey2003** [comment]
Will remove it

**Comment by arjunpandey2003** [comment]
@vaibhavsingh97 replaced the link

File: index.html [hunk 2]
```diff
@@ -666,7 +669,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://datacharmer.org/"><i class="icon fa fa-external-link"></i></a>
-                                             <a target="default" href="https://github.com/datacharmer"><i class="icon fa fa-github"></i></a>											<a target="default" href="https://twitter.com/yacy_search"><i class="icon social_twitter"></i></a>
+                                                         				<a target="default" href="https://github.com/datacharmer"><i class="icon fa fa-github"></i></a>											<a target="default" href="https://twitter.com/yacy_search"><i class="icon social_twitter"></i></a>
```
Comment (vaibhavsingh97) [inline index.html]: Please indent properly

**Comment by arjunpandey2003** [comment]
@vaibhavsingh97 done

**Comment by arjunpandey2003** [comment]
I have indented more

**Comment by arjunpandey2003** [comment]
Please review @vaibhavsingh97

**Comment by arjunpandey2003** [comment]
@0x48piraj @saarthakchats thanks!

**Comment by arjunpandey2003** [comment]
@mariobehling please review and merge

**Pull request closed without merging**